### PR TITLE
LPS-172655 | Test Fix | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Element is not present at Segments with AC sync

Hi! :)
This pr is corresponding to this ticket: [LPS-172655](https://issues.liferay.com/browse/LPS-172655)
Is related to [LPS-160028](https://issues.liferay.com/browse/LPS-160028)

### Problem:
The sync time between AC and DXP is one minute. The sync is not successfully performed in time for the tests to assert.

### Fix:
Add a while inside the test with a maximum of interactions so that the sync is done in time for validation.

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ACSegmentsInDXP.testcase [hunk 1]
```diff
@@ -206,15 +209,11 @@ definition {
 				category = "People",
 				portlet = "Segments");
 
-			AssertElementPresent(
-				key_tableEntry = "Dynamic Segment Test",
-				locator1 = "LexiconTable#TABLE_ENTRY_ELLIPSIS");
-		}
-
-		task ("Delete Dynamic Segment Test") {
-			Segmentation.deleteSegment(entry = "Dynamic Segment Test");
```
Comment (rafaellajordao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ACSegmentsInDXP.testcase:215]: Hey @CarlosBrichete , currently it is not possible to delete a segment that was created in AC by the DXP side, correct?

Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ACSegmentsInDXP.testcase:215]: I think you can delete the appearance of the segment in DXP, but the segment is not deleted from AC. In fact, the segment will appear again in DXP when the sync cycle time finishes.

**Comment by CarlosBrichete** [comment]
Hey Rafa! LGTM! Let's wait for the tango suite to be finished to check that the tests are passing

**Comment by CarlosBrichete** [comment]
Hey Rafa, these tests are still failing can you take a look?
* ACSegmentsInDXP#RedirectToSegmentPageErrorAC
* ACSegmentsInDXP#RedirectToSegmentPageAC

**Comment by rafaellajordao** [comment]
> Hey @rafaellajordao , these tests are still failing can you take a look?
> 
> * ACSegmentsInDXP#RedirectToSegmentPageErrorAC
> * ACSegmentsInDXP#RedirectToSegmentPageAC

I'll check!!

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:forward
